### ISSUE TITLE
\hangindentX for footnotes with more than one paragraph 

### ISSUE THREAD

**Issue by tweh** [maieul/ledmac#389]
At the moment the `\hangindent` command only works for the first paragraph of a familiar:

```latex
\documentclass{scrbook}

\usepackage[parapparatus]{reledmac}

\hangindentX{4em}

\usepackage{lipsum}

\begin{document}
   Lipsum\footnoteA{\lipsum[1-2]} \lipsum[1]
\end{document} 
```

(I’ll make a pull request with a possible fix …)

**Comment by maieul** [comment]
yes, tex \hangindent work only for the next paragraph.

I thinks \everypar should solve it for your pull request

**Comment by tweh** [comment]
Yep that exactly, what I’m adding to your code right now … :-)

**Comment by tweh** [comment]
Ok … I send a pull request. But I’m not sure if I’m using the `git` stuff/tools right … am I?

**Comment by maieul** [comment]
pull request is for that ;-)

**Comment by tweh** [comment]
Yep thats what I thought, but I was unsure about the forging. Since I forked your repo and send the pull request to my fork … but that seems to be the way to go :-)

**Comment by maieul** [comment]
no, you should fork my repository, create a branch, push it in your repository, and send a pull request from this branch to my repository.
for now, I can't merge (you have done in past pull request)

**Comment by tweh** [comment]
hm … as far as I can tell I send a pull request but it seems like to my fork: https://github.com/tweh/ledmac/pull/1 how can I send it to you?

**Comment by maieul** [comment]
https://help.github.com/articles/using-pull-requests/

please send it in my branch "2.1.3-2.2.1"

**Comment by tweh** [comment]
Just tried it again … was it right this time?

**Comment by maieul** [comment]
fix par #390
